### PR TITLE
fix(apple): Mention not creating TTID/TTFD spans

### PR DESCRIPTION
**Pull request by philipphofmann**
## DESCRIBE YOUR PR

Explain why the SDK doesn't create TTID/TTFD spans when the app is in the background.

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [x] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

### CODE REVIEW

File: docs/platforms/apple/common/tracing/instrumentation/automatic-instrumentation.mdx [hunk 1]
```diff
@@ -333,13 +333,13 @@ _(New in version 8.4.0)_
...
 
-The span finishes after the view appeared on the screen.
+![](./img/ttid-ttfd-concept.png)
 
-The following chart shows how time to initial display (TTID) and [time to full display (TTFD)](#time-to-full-display) correlate to transitions between activities:
+Since Cocoa SDK version 8.33.0, the SDK doesn't create time to initial display (TTID) and [time to full display (TTFD)](#time-to-full-display) for UIViewControllers presented in the background because the logic requires UI frames to be drawn.
 
-![](./img/ttid-ttfd-concept.png)
+The SDK doesn't create time to initial display (TTID) and [time to full display (TTFD)](#time-to-full-display) spans for UIViewControllers presented after receiving the [UIApplicationDidEnterBackgroundNotification](https://developer.apple.com/documentation/uikit/uiapplicationdidenterbackgroundnotification), because the logic requires UI frames to be drawn.
```
Comment (kahest) [inline docs/platforms/apple/common/tracing/instrumentation/automatic-instrumentation.mdx:342]: IIUC this was always the case, so this is not something we changed in 8.33.0, correct? if so, I'd put this (more generic) information above the note about the change in 8.33.0

**Review by kahest — APPROVED** [review]
left a question for clarification, otherwise LGTM, thanks!

File: docs/platforms/apple/common/tracing/instrumentation/automatic-instrumentation.mdx (hunk 1 above)
Comment (philipphofmann) [reply docs/platforms/apple/common/tracing/instrumentation/automatic-instrumentation.mdx:342]: No, we changed this in 8.33.0. In previous versions, the SDK created long-lasting spans for TTID.

Comment (philipphofmann) [reply docs/platforms/apple/common/tracing/instrumentation/automatic-instrumentation.mdx:342]: Ah ups, forgot to mention 8.33.0 here. Now I get it.

File: docs/platforms/apple/common/tracing/instrumentation/automatic-instrumentation.mdx [hunk 2]
```diff
@@ -353,7 +351,8 @@ _(New in version 8.4.0)_
 
 By adding a span for a view controller when it's loaded, time to full display (TTFD) provides insight into how long it takes your view controller to launch and load all of its content. The SDK sets the span operation to `ui.load.full-display` and the span description to the view controllers' name, followed by `full display` - for example, `MainActivity full display`.
```
Comment (vivianyentran) [inline docs/platforms/apple/common/tracing/instrumentation/automatic-instrumentation.mdx:352]:
```suggestion
By adding a span for a view controller when it's loaded, time to full display (TTFD) provides insight into how long it takes your view controller to launch and load all of its content. The span starts when the view of a view controller is loaded, and there is no other view controller transaction happening at the moment. The SDK sets the span operation to `ui.load.full-display` and the span description to the view controllers' name, followed by `full display` - for example, `MainActivity full display`.
```

**Review by vivianyentran — APPROVED** [review]

**Pull request merged**